pkg/tasks/trusted_ca.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2020 The Cluster Monitoring Operator Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tasks
+
+import (
+	"time"
+
+	"github.com/openshift/cluster-monitoring-operator/pkg/client"
+	"github.com/openshift/cluster-monitoring-operator/pkg/manifests"
+	"github.com/pkg/errors"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+type caBundleSyncer struct {
+	prefix  string
+	client  *client.Client
+	factory *manifests.Factory
+}
+
+func (cbs *caBundleSyncer) syncTrustedCABundle(trustedCA *v1.ConfigMap) (*v1.ConfigMap, error) {
```

Comment:
To me in this package we only have very clear separated tasks per file, this seems like a helper function shared by the tasks wdyt if that should go in the client or manifests package instead? Or a new package like we have with strings package?

Reply:
It can't really go in `pkg/client` or `pkg/manifests` because it depends on both. I can create a new package but I felt that since it's only used in `pkg/tasks`, it could live there. Maybe it's easier to differentiate from the other files if I rename it `helpers.go`?

Reply:
> Maybe it's easier to differentiate from the other files if I rename it helpers.go

That works for me as well, either helpers.go or utils.go.

Reply:
I've picked `helpers.go`.